integration/testnet/network.go

```diff
@@ -61,6 +61,8 @@ const (
 	// DefaultExecutionRootDir is the default directory for the execution node
 	// state database.
 	DefaultExecutionRootDir = "/exedb"
+	// DefaultExecutionDataServiceDir for the execution data service blobstore.
+	DefaultExecutionDataServiceDir = "/data/execution_data"
```

Comment:
I remember seeing `execution_data_blobstore` above, are the two values suppose to be the same? might make sense to name them the same if they have the same purpose, to make debugging easier